packages/node_modules/@ciscospark/widget-space/src/actions.js

```diff
@@ -5,6 +5,7 @@ import messages from './messages';
 
 
 export const FETCH_SPACE_ERROR = `widget-space/FETCH_SPACE_ERROR`;
+export const STORE_TO_PERSON = `widget-space/STORE_TO_PERSON`;
```

Comment:
Haha... yea, we need to actually refactor the `redux-module-users`. One that happens we can actually just point to the stored user in whatever widgets we build.